tests/ci_tests/code_formatter/crewai/test_crewai_agents.py

```diff
@@ -0,0 +1,310 @@
+
+    def test_agent_has_no_tools(self):
+        """Test that the senior engineer agent has no tools"""
+        mock_langfuse_client = mock.MagicMock()
+
+        mock_goal_prompt = mock.MagicMock()
+        mock_goal_prompt.prompt = "goal"
+        mock_backstory_prompt = mock.MagicMock()
+        mock_backstory_prompt.prompt = "backstory"
+
+        mock_langfuse_client.get_prompt.side_effect = [
+            mock_goal_prompt,
+            mock_backstory_prompt,
+            "template",
+            "template",
+        ]
+
+        agent_map = agents.create_code_formatter_agents(mock_langfuse_client)
+        agent = agent_map["senior_engineer"]
+
+        assert agent.tools == []
+
+    def test_agent_does_not_delegate(self):
+        """Test that the agent has delegation disabled"""
+        mock_langfuse_client = mock.MagicMock()
+
+        mock_goal_prompt = mock.MagicMock()
+        mock_goal_prompt.prompt = "goal"
+        mock_backstory_prompt = mock.MagicMock()
+        mock_backstory_prompt.prompt = "backstory"
+
+        mock_langfuse_client.get_prompt.side_effect = [
+            mock_goal_prompt,
+            mock_backstory_prompt,
+            "template",
+            "template",
+        ]
+
+        agent_map = agents.create_code_formatter_agents(mock_langfuse_client)
+        agent = agent_map["senior_engineer"]
+
+        assert agent.allow_delegation is False
+
+    def test_agent_is_verbose(self):
+        """Test that the agent has verbose mode enabled"""
+        mock_langfuse_client = mock.MagicMock()
+
+        mock_goal_prompt = mock.MagicMock()
+        mock_goal_prompt.prompt = "goal"
+        mock_backstory_prompt = mock.MagicMock()
+        mock_backstory_prompt.prompt = "backstory"
+
+        mock_langfuse_client.get_prompt.side_effect = [
+            mock_goal_prompt,
+            mock_backstory_prompt,
+            "template",
+            "template",
+        ]
+
+        agent_map = agents.create_code_formatter_agents(mock_langfuse_client)
+        agent = agent_map["senior_engineer"]
+
+        assert agent.verbose is True
+
+    def test_langfuse_error_handling(self):
+        """Test that Langfuse errors are propagated"""
+        mock_langfuse_client = mock.MagicMock()
+        mock_langfuse_client.get_prompt.side_effect = Exception("Cannot fetch prompts from Langfuse")
+
+        with pytest.raises(Exception, match="Cannot fetch prompts from Langfuse"):
+            agents.create_code_formatter_agents(mock_langfuse_client)
+
+
+class TestCreatePRReviewerAgent:
+    """Tests for create_pr_reviewer_agent function"""
+
+    def test_returns_dict_with_reviewer(self):
+        """Test that the function returns a dict with 'pull_request_reviewer' key"""
+        mock_langfuse_client = mock.MagicMock()
+
+        result = agents.create_pr_reviewer_agent(mock_langfuse_client)
+
+        assert isinstance(result, dict)
+        assert "pull_request_reviewer" in result
```

Comment:
Added a docstring to describe the class's purpose.
```suggestion
class TestCreatePRReviewerAgent:
```